apps/bridge/components/WormholeToggle.tsx

```diff
@@ -0,0 +1,22 @@
+import { useTranslation } from '@pancakeswap/localization'
+import { Flex, Text, Toggle } from '@pancakeswap/uikit'
+import { useRouter } from 'next/router'
+import { useEnableWormholeMainnet } from 'state/wormhole/enableTestnet'
+import { isDev } from './WormHole/constants'
+
+export function WormholeEnvToggle() {
+  const [enableMainnet, setEnableMainnet] = useEnableWormholeMainnet()
+  const { t } = useTranslation()
+  const { pathname } = useRouter()
+
+  if (pathname !== '/wormhole' || !isDev) return null
```

Comment:
not a good practice to check with pathname. It'll break everything if we simply just move the wormhole page to another path